composition/src/client_component.cpp

```diff
@@ -48,22 +48,23 @@ Client::Client()
 : Node("Client")
 {
   client_ = create_client<example_interfaces::srv::AddTwoInts>("add_two_ints");
-  timer_ = create_wall_timer(1s, std::bind(&Client::on_timer, this));
+  timer_ = create_wall_timer(2s, std::bind(&Client::on_timer, this));
```

Comment:
Can you add a comment here describing why the timer must have a larger timeout than the wait for service call below.

Reply:
Yep: 894d91851531fe418113fd30f6f0f652977ef3cf